barretenberg/cpp/src/barretenberg/dsl/acir_proofs/goblin_acir_composer.cpp

```diff
@@ -15,10 +15,6 @@ void GoblinAcirComposer::create_circuit(acir_format::AcirFormat& constraint_syst
 
     // Populate constraints in the builder via the data in constraint_system
     acir_format::build_constraints(builder_, constraint_system, true);
-
```

Comment:
Its debatable whether we should remove the ops here. In the case of UGH, removing them is fine since eventually we'd like a world where we dont NEED goblin ops to make a valid proof. This one, however, is a Goblin specific test so it would defeat the purpose of the test to not have any ops. So when/if we get rid of the `add_gates_to_ensure..`, this test would be quietly made useless. I think we should continue to explicitly add ops here in one way or another, with a comment explaining why, so someone who thinks they're clever doesn't go and remove them later. What do you think?